### ISSUE TITLE
70_CanOverEthernet.pm wrong delimter

### ISSUE THREAD

**Issue by jaghatei** [delMar43/FHEM#13]
I had to adjust line 170 to make the module sending analog data via COE to CMI.
Problematic is the usage of semicolon as a delimiter between value and type causing $type always to be uninitialized which prevents any sending of analog data.

`my ( $index, $value, $type ) = split /[=;]/, $valuesAndTypes[$i];`
set coe sendDataAnalog 192.168.178.44 4 1=9.1;1
=> no data sent.

Once delimiter is changed to double point it works:

`my ( $index, $value, $type ) = split /[=:]/, $valuesAndTypes[$i];`
set coe sendDataAnalog 192.168.178.44 4 1=9.1:1
=> analog data sent properly

I cannot tell if this is a general issue or only specific for my setup ( RPI with Debian buster and Perl v5.28.1)

**Comment by MartinGoodwell** [comment]
Correct, the semicolon is kinda problematic, as you'd need to use two of them, so FHEM doesn't treat it as a separate command.
Bad design thinking from my side.
I'll add your change as an option, so the module will allow both, colon and semi-colon, as separators.
Thanks for your input
